src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,110 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    public static final int ZERO = 0;
```

Comment:
Zero might not be included in constants

Reply:
Agree with previous comment, this constant is redundant.